src/test/java/org/corfudb/infrastructure/LayoutServerTest.java

```diff
@@ -148,8 +155,8 @@ public void checkLayoutPersisted()
         LayoutServer s1 = getDefaultServer(serviceDir);
         bootstrapServer(TestLayoutBuilder.single(9000));
         Layout l100 = TestLayoutBuilder.single(9000);
-        l100.setEpoch(100);
-        sendMessage(new LayoutRankMsg(null, 100, CorfuMsgType.LAYOUT_PREPARE));
+        //l100.setEpoch(100);
```

Comment:
I have to admit I got lost here, what are we checking from here on? comments would help